hil/api.py

```diff
@@ -119,6 +121,7 @@ def project_detach_node(project, node):
     """
     project = _must_find(model.Project, project)
     get_auth_backend().require_project_access(project)
+    node_label = node
```

Comment:
just a suggestion, take it or leave it.
 don't really need to keep a copy here. we could just use `node.label` in the code later after we get Node object.

Reply:
Yeah I definitely agree with that.  I'll remove node_label.